packages/admin/src/Navigation/NavigationItem.php

```diff
@@ -16,19 +16,30 @@ class NavigationItem
 
     protected ?string $badge = null;
 
+    protected ?string $badgeColor = null;
+
     protected bool $shouldOpenUrlInNewTab = false;
 
     protected ?int $sort = null;
 
     protected string | Closure | null $url = null;
 
-    final public function __construct()
+    final public function __construct(?string $label = null)
+    {
+        if (filled($label)) {
```

Comment:
Why the `filled` check here and not for groups?

Reply:
NavigationItem labels are required, NavigationGroup labels are nullable. It's a BC if we force users to pass a label in the NavigationItem constructor so it needs to be nullable